packages/babel-preset/src/transformPlugin.ts

```diff
@@ -166,6 +266,8 @@ export const transformPlugin = declare<Partial<BabelPluginOptions>, PluginObj<Ba
               pluginOptions,
             );
 
+            state.cssResetEntries ??= {};
```

Comment:
Please do this in `pre()` if needed

Reply:
I don't mind initializing in `pre` but the types won't change, and you need to have `if` assertions. Is there any consequence of using nullish coalescing assignment?